### ISSUE TITLE
Unit and functional test coverage

### ISSUE THREAD

**Issue by ShahanaFarooqui** [ShahanaFarooqui/RTL#197]
Unit and functional test coverage

**Comment by saiy2k** [comment]
Hi @ShahanaFarooqui .
Shall I take this up? Writing unit/functional test for the whole application would be huge task. But I can start with a single screen/module.

Can you tell me any specific modules / components to start with?

**Comment by ShahanaFarooqui** [comment]
@saiy2k This is a good issue, but definitely a huge one. However, as you said, you can just pick one component at a time and keep moving. I added some sample test cases to showcase on how test cases can be written, for example: `/app/lnd/transactions/send-payment-modal`, `./app/store`, etc. 

We can set up a call if you are unclear about the flow of the testing process or about how to find and write test cases, etc.

**Comment by saiy2k** [comment]
@ShahanaFarooqui Yes. That would be helpful. Can you please schedule a Google meet/Zoom call to saiy2k@gmail.com, anytime / timezone on Saturday/Sunday.

I will also have a look at existing test cases before we connect.

Thanks.

**Comment by ShahanaFarooqui** [comment]
@saiy2k Removed your email info from the thread.  Will communicate thru Twitter DM or email soon.
